jupyterhub/singleuser.py

```diff
@@ -48,10 +48,9 @@ def verify_token(self, cookie_name, encrypted_cookie):
         hub_api_url = self.settings['hub_api_url']
         hub_api_key = self.settings['hub_api_key']
         r = requests.get(url_path_join(
-            hub_api_url, "authorizations/cookie", cookie_name,
```

Comment:
Is the text of encrypted_cookie guaranteed to be url-safe and/or does requests automatically escape it intelligently?